### ISSUE TITLE
Possible Crashes on iOS 18.1 Beta with Continuous Tracking Mode

### ISSUE THREAD

**Issue by klimentij** [owntracks/ios#816]
- **Device:** iPhone 11 (MHCX3LL/A)  
- **iOS Version:** 18.1 beta (22B5069a)  
- **App Version:** OwnTracks 18.0.4 / en_AR  

Thank you for your fantastic work on this app! I've been using OwnTracks in **Move mode**, sending location updates every 30 seconds, 24/7, for several months now. However, around the time I updated my phone to the iOS 18 beta, I started noticing occasional issues that might indicate the app is crashing. I'm not sure if I had this issue on iOS 17

**Observed Behavior:**  
1. **Missing location updates:** The frontend stops receiving new locations after some time.  
2. **Longer app startup:** When I manually open the app, it takes longer to launch, showing the welcome logo screen as if it were starting fresh.  

**Note:** Unfortunately, I couldn't retrieve any crash logs from these events. It seems they might have been overwritten or erased.  

**Possible Causes:**  
- This could be related to my recent switch from iOS 17 to iOS 18 beta, but I can't confirm it with certainty.  

**Workaround:**  
To prevent missing location data, I manually reopen the app every morning. However, if it crashes during the day, I lose the track for that period, which is not ideal.

I greatly appreciate the time and effort you put into developing and maintaining this app—it’s been incredibly useful for my tracking needs! If there’s anything I can do to help troubleshoot or provide more details, please let me know.  

Thank you again for your amazing work!

**Comment by ckrey** [comment]
Thanks for using OwnTracks.

I don't have access to iOS 18.1 beta. I have not received any crash logs from an iPhone 11 device running 18.1.
I received some crashes from other devices running 18.1 related to network.

Do you have [Share iPhone & Watch Analytics](https://support.apple.com/en-us/108971) enabled on your device? This might be useful if you are testing betas.

The OwnTracks seems to work well on the official iOS 18.0.1. If you don't want to risk loosing data, consider downgrading to the latest official iOS release.

**Comment by klimentij** [comment]
Yes, I'm on 18.3 (22D63) (public, not beta), and it still crashes silently once in a few days. I have an automation that reminds me to manually start the app twice a day.

**Comment by ckrey** [comment]
A public beta of version 18.3.5 addressing the problem is available via TestFlight:
https://testflight.apple.com/join/k2iqRtjH

**Comment by klimentij** [comment]
Thank you! Switched to it, I'll let you know if I have any issues.

By the way, I open-sourced my tracking setup where I use your app: https://github.com/klimentij/NerdTracker 

@ckrey
